programs/voter-stake-registry/src/instructions/create_deposit_entry.rs

```diff
@@ -81,10 +80,10 @@ pub fn create_deposit_entry(
 
     let start_ts = clock_unix_timestamp();
     *d_entry = DepositEntry::default();
-    d_entry.delegate = delegate;
     d_entry.is_used = true;
     d_entry.voting_mint_config_idx = mint_idx as u8;
     d_entry.amount_deposited_native = 0;
+    d_entry.delegate_mining = *ctx.accounts.voter_authority.key;
```

Comment:
this doesn't look like user's mining account.

Reply:
It isn't. Just a blank value, it could've been `Default` instead.